ieeeuoft/makeuoft-site-2022
# Copy over styles from MakeUofT 2021

## Overview

- Resolves IEEE-157
- Copies over styles and graphics from MakeUofT 2021. I pretty much just diffed every relevant file between the two repos (with Pycharm), and pulled in all the necessary changes. Some things were changed slightly to keep with the same page structure that we use in the template now, which should make it easier to use in the future.
- All of the content is still from 2021 - Logistics will need to go through it and get us the latest content for this year

## Unit Tests Created

- None, all visual

## Steps to QA

- Clone the repo, run the site, and test out all the main pages to make sure they look right. Make sure to do `yarn run scss` from first.
